scripts/build/libc/glibc.sh

```diff
@@ -72,6 +72,7 @@ do_libc_backend() {
     local multilib
     local multi_dir
     local multi_flags
+    local target
```

Comment:
No, *this* function sets ${target}, ${libc_target} is used in the called function (do_libc_backend_once).